src/docs/guides/21-how-to-setup-and-use-stripe/index.md

```diff
@@ -0,0 +1,112 @@
+started with Stripe?". You can click "Skip for now" link in the bottom
+of the page to get directly to Stripe dashboard. Remember to confirm
+your email address after the registration.
+
+As you will receive money from your users via your Stripe account, you
+have to provide some details such as your address and your bank account.
+In the Stripe dashboard click the Activate your account link in the left
+menu and complete all the fields according to the instructions. The
+activation form varies based on your country.
+
+![Activate Stripe account](./activate-account.png)
+
+## 2. Enable Stripe Connect in your platform
+
+Sharetribe uses the Stripe Connect features with Custom Accounts.
+
+- Click the Connect left menu item.
+- Click the Get started button.
+- Now you should see your Connect dashboard. From the left sidebar, go
+  to Connect / Settings
+- Make sure that "Custom accounts are available for your platform." is
+  shown.
+
+![Stripe Connect](./connect.png)
+
+## 3. Get your API keys from Stripe and add them to your Sharetribe marketplace
+
+- Click "Developers" → "API Keys" in the left menu item.
+- In the section "Standard API keys" you will see two keys: publishable
+  key and secret key. The publishable key (with prefix pk*) is one used
+  in frontend application e.g. FTW and secret key (with prefix sk*) is
+  the one you need to add to Flex Console. If you want to use test data
+  make sure the value of the key is eg. pk_test_somethinghere and not
+  pk_live_somethinghere
+
+**Note:** If you want to use test data in development make sure that
+"View test data" toggle on. This way no real money will be used. In
+production make sure that the toggle is off.
+
+![Get API keys from Stripe](./api-keys.png)
+
+## 4. Add your Stripe secret API key to Flex Console
+
+- Log in to the Flex Console and go to "Build" -> "Payments"
+- In the section "Stripe configuration" paste your secret key to "Stripe
+  secret key" field and save the changes
+
+![Add Stripe secret key to Flex Console](./add-stripe-to-console.png)
+
+## 5. Add you Stripe publishable key to FTW
+
+If you are using Flex Template for Web you need to add the Stripe
+publishable key to your `.env` file. You can do this by running
+`yarn run config` or editing the file directly in a text editor.
+
+Read more about configurations in FTW from
+[Getting started with FTW](/getting-started-with-ftw/#configuration)
+
+## 6. Testing the payments
+
+**Note:** When testing Stripe, make sure you are using the test API
+keys. To ensure that make sure the keys have prefix sk_test and pk_test.
+When checking the Stripe dashboard, make sure "View test data" toggle is
+on!
+
+Stripe provides test numbers for bank accounts, credit cards etc.
+Specific numbers are also provided for triggering different conditions
+(e.g. card declined or expired) so testing different error scenarios is
+also possible. See Stripe documentation for
+[testing with Connect](https://stripe.com/docs/connect/testing).
```

Comment:
This is about testing payouts. Even more important link is test credit card numbers which can be used in CheckoutPage: https://stripe.com/docs/testing#cards

Reply:
I was also thinking if it would make sense to add one more screenshot here. Something like:
![screenshot 2019-02-04 at 16 53 36](https://user-images.githubusercontent.com/717315/52215818-8d78ee00-289d-11e9-9b26-15f53c988f35.png)

Reply:
What comes to the bank account numbers, it might be good to tell a bit more about how those are mapped.

E.g. Hongkong routing number consists of clearing code and branch code.

We ask them separately:
![screenshot 2019-02-04 at 17 46 10](https://user-images.githubusercontent.com/717315/52219352-24957400-28a5-11e9-83c1-0cbbbd34f822.png)

But Stripe test account data has mapped them together to routing number:
![screenshot 2019-02-04 at 17 46 45](https://user-images.githubusercontent.com/717315/52219468-5dcde400-28a5-11e9-8f04-fb4afd333a11.png)